source/api/image/2.1/compliance.md

```diff
@@ -46,21 +48,25 @@ In the tables below "![required][icon-req]" indicates that support is _REQUIRED_
 | `square`    | `regionSquare` | ![optional][icon-opt] | ![optional][icon-opt] | ![optional][icon-opt] |
 {: .api-table}
 
-### Size
-
-| Syntax      | Feature Name        | Level 0 | Level 1 | Level 2  |
-|:------------|:--------------------|:-------:|:-------:|:--------:|
-| `full`      |                     | ![required][icon-req]      | ![required][icon-req]      | ![required][icon-req]       |
-| w,h         | `sizeByWhListed` | ![required][icon-req]      | ![required][icon-req]      | ![required][icon-req]       |
-| w,          | `sizeByW`         | ![optional][icon-opt]      | ![required][icon-req]      | ![required][icon-req]       |
-| ,h          | `sizeByH`         | ![optional][icon-opt]      | ![required][icon-req]      | ![required][icon-req]       |
-| pct:x       | `sizeByPct`       | ![optional][icon-opt]      | ![required][icon-req]      | ![required][icon-req]       |
-| w,h         | `sizeByForcedWh` | ![optional][icon-opt]      | ![optional][icon-opt]      | ![required][icon-req]       |
-| !w,h        | `sizeByWh`        | ![optional][icon-opt]      | ![optional][icon-opt]      | ![required][icon-req]       |
-|             | `sizeAboveFull`   | ![optional][icon-opt]      | ![optional][icon-opt]      | ![optional][icon-opt]       |
+See also note under [Size][size] about combinations of Size and Region that may not be supported.
+
+### 3.2 Size
+
+| Syntax | Feature Name     | Level 0 | Level 1 | Level 2  |
+|:-------|:-----------------|:-------:|:-------:|:--------:|
+| `full` |                  | ![required][icon-req] | ![required][icon-req] | ![required][icon-req] |
+| w,h    | `sizeByWhListed` | ![required][icon-req] | ![required][icon-req] | ![required][icon-req] |
+| w,     | `sizeByW`        | ![optional][icon-opt] | ![required][icon-req] | ![required][icon-req] |
+| ,h     | `sizeByH`        | ![optional][icon-opt] | ![required][icon-req] | ![required][icon-req] |
+| pct:x  | `sizeByPct`      | ![optional][icon-opt] | ![required][icon-req] | ![required][icon-req] |
+| w,h    | `sizeByForcedWh` | ![optional][icon-opt] | ![optional][icon-opt] | ![required][icon-req] |
+| !w,h   | `sizeByWh`       | ![optional][icon-opt] | ![optional][icon-opt] | ![required][icon-req] |
+|        | `sizeAboveFull`  | ![optional][icon-opt] | ![optional][icon-opt] | ![optional][icon-opt] |
 {: .api-table}
 
-### Rotation
+Note that servers may express limits on the sizes available for an image with the optional `maxWidth`, `maxHeight` and/or `maxArea` [Profile Description properties][profile]. Servers are compliant provided they support the forms of the Size parameter shown above for image sizes up to the limits specified. Clients _SHOULD NOT_ assume that Region and Size parameter combinations such as `/full/full/` will be supported. 
+
```

Comment:
And should we be telling clients what to do at all? Maybe we leave the warning text in place but remove the RFC _SHOULD NOT_?